client/components/AccountCreation/AccountCreation.jsx

```diff
@@ -22,13 +23,17 @@ function AccountCreation(props) {
     //Whenever the submit button is clicked, this checks to make sure the passwords match and calls another func
     const handleSubmit = (e) => {
         e.preventDefault();
-        if(formData.password !== formData.confirmPassword) {
-            alert("YOUR PASSWORDS DO NOT MATCH");
-        }
-        else if(checkPasswordRequirements(formData.password) && checkUsernameRequirements(formData.username)) {
-            //TODO send the formData to the backend
-            console.log("Hey this code works");
-            props.close();
+        try {
+            if(formData.password !== formData.confirmPassword) {
+                alert("YOUR PASSWORDS DO NOT MATCH");
+            }
+            else if(checkPasswordRequirements(formData.password) && checkUsernameRequirements(formData.username)) {
+                //TODO send the formData to the backend
+                console.log("Hey this code works");
```

Comment:
can you remove this log while you're here? it should've been caught in user login front end review

Reply:
Required for a test at here: 
https://github.com/PistolRcks/web-puzzle-project/blob/5f6151751a2ddc72e09c0ad664d49e38a503db96/tests/client/AccountCreation.test.js#L72 
Perhaps a better way of writing the test is in order? Either way, rewriting the test is out of scope. Resolving.